docs/jwst/pathloss/reference_files.rst

```diff
@@ -52,3 +56,80 @@ as the behavious or each slice is considered identical.
+CRPIX1  1.0        Reference pixel in fastest dimension
+CRVAL1  -0.5       Coordinate value at this reference pixel
+CDELT1  0.05       Change in coordinate value for unit change in index
+CTYPE1  'UNITLESS' Type of physical coordinate in this dimension
+======= ========== =========================================================================================
+
+This dimension expresses the decenter along the dispersion direction for a point source
+
+======= ========== =========================================================================================
+CRPIX2  1.0        Reference pixel in fastest dimension
+CRVAL2  -0.5       Coordinate value at this reference pixel
+CDELT2  0.05       Change in coordinate value for unit change in index
+CTYPE2  'UNITLESS' Type of physical coordinate in this dimension
+======= ========== =========================================================================================
+
+This dimension expresses the decenter along the direction perpendicular to the dispersion for a point source
+
+======= ========== =========================================================================================
+CRPIX3  1.0        Reference pixel in fastest dimension
+CRVAL3  6.0E-7     Coordinate value at this reference pixel
+CDELT3  2.35E-7    Change in coordinate value for unit change in index
+CTYPE3  'Meter'    Type of physical coordinate in this dimension (should be 'WAVELENGTH')
+======= ========== =========================================================================================
+
+This dimension expresses the change of correction with wavelength
+
+The NIRISS_SOSS reference file should also have WCS components, but their
+interpretation is different from those in the NIRSPEC reference file:
+
+======= ===================== =========================================================================================
+Keyword Value                 Comment
+======= ===================== =========================================================================================
+CRPIX1  5.0                   Reference pixel in fastest dimension
+CRVAL1  5.0                   Coordinate value at this reference pixel
+CDELT1  1.0                   Change in coordinate value for unit change in index
+CTYPE1  'PIXEL'               Type of physical coordinate in this dimension
+======= ===================== =========================================================================================
+
+This dimension expresses the decenter along the dispersion direction for a point source
+
+======= ===================== =========================================================================================
+CRPIX2  9.0                   Reference pixel in fastest dimension
+CRVAL2  245.304               Coordinate value at this reference pixel
+CDELT2  0.1                   Change in coordinate value for unit change in index
+CTYPE2  'Pupil Wheel Setting' Type of physical coordinate in this dimension
```

Comment:
Is that an allowable CTYPE value?

Reply:
I don't think there's a fixed set of possibilities for CTYPE, as it could pretty much refer to anything.

Reply:
I see spaces in the value string and I cringe, although strictly speaking it's allowed.